src/main/java/homework2/Ticket.java

```diff
@@ -37,26 +38,94 @@ public Ticket(String id, String concertHall, int eventCode, boolean isPromo,
         this.price = price;
     }
 
+
+    public Sector getSector() {
+        return sector;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public String getConcertHall() {
+        return concertHall;
+    }
+
+    public int getEventCode() {
+        return eventCode;
+    }
+
+    public boolean isPromo() {
+        return isPromo;
+    }
+
+    public float getMaxWeight() {
+        return maxWeight;
+    }
+
+    public void setTime(String time) {
+        this.time = time;
+    }
+
+    public void setSector(Sector sector) {
+        this.sector = sector;
+    }
+
     private String getTime() {
-        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd-MM-yy HH:mm");
+        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd-MM-yy HH:mm:ss");
         String dateTime = dtf.format(LocalDateTime.now());
         return dateTime;
     }
 
-    public String getId() {
-        return id;
+    public static void shared(int ticketId, String phoneNumber) {
+        System.out.println("Ticket #" + ticketId + " was shared by phone " + phoneNumber);
     }
 
-    public Sector getSector() {
-        return sector;
+
+    public static void shared(int ticketId, String phoneNumber, String email) {
+        System.out.println("Ticket #" + ticketId + " was shared by phone " + phoneNumber + " and email " + email);
```

Comment:
Why this methods are static ?
Just this.id you can use

Reply:
Remove for parameter, and add in sout

Reply:
Red because you need call
ticket.share(...)

Reply:
a ok. I thinked there is something new. that i write this.id and non static method wil be called)))